apps/src/assetManagement/AssetManager.jsx

```diff
@@ -147,15 +147,6 @@ module.exports = React.createClass({
       );
     }
 
-    var title = this.props.assetChosen ?
-        <p className="dialog-title">Choose Assets</p> :
-        <p className="dialog-title">Manage Assets</p>;
-
-    return (
-      <div className="modal-content" style={{margin: 0}}>
-        {title}
-        {assetList}
-      </div>
-    );
+    return assetList;
```

Comment:
Could take this opportunity to convert AssetManager to the form listed here: https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#js-module-exports. I think converting all components to use that form would make it so that the warning about key props would tell you which component had the issue, instead of just saying `exports`